### ISSUE TITLE
Ticket generation crashes when event is deactivated

### ISSUE THREAD

**Issue by mihob** [verbb/Events#174]
### Describe the bug

The ticket generation throws an error if the event is deactivated.

```
Error: Call to a member function getType() on null
#18 /app/vendor/verbb/events/src/elements/Ticket.php(483): verbb\events\elements\Ticket::beforeSave
#17 /app/vendor/craftcms/cms/src/services/Elements.php(3503): craft\services\Elements::_saveElementInternal
#16 /app/vendor/craftcms/cms/src/services/Elements.php(1303): craft\services\Elements::saveElement
#15 /app/vendor/verbb/events/src/elements/Event.php(1018): verbb\events\elements\Event::updateTickets
#14 /app/vendor/verbb/events/src/elements/Event.php(1125): verbb\events\elements\Event::afterPropagate
#13 /app/vendor/craftcms/cms/src/services/Elements.php(3818): craft\services\Elements::craft\services\{closure}
#12 /app/vendor/craftcms/cms/src/services/Elements.php(1217): craft\services\Elements::ensureBulkOp
#11 /app/vendor/craftcms/cms/src/services/Elements.php(3578): craft\services\Elements::_saveElementInternal
#10 /app/vendor/craftcms/cms/src/services/Elements.php(1303): craft\services\Elements::saveElement
#9 /app/vendor/craftcms/cms/src/controllers/ElementsController.php(1260): craft\controllers\ElementsController::actionSave
#8 [internal](0): call_user_func_array
#7 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): yii\base\InlineAction::runWithParams
#6 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\Controller::runAction
#5 /app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Module::runAction
#4 /app/vendor/craftcms/cms/src/web/Application.php(350): craft\web\Application::runAction
#3 /app/vendor/craftcms/cms/src/web/Application.php(649): craft\web\Application::_processActionRequest
#2 /app/vendor/craftcms/cms/src/web/Application.php(312): craft\web\Application::handleRequest
#1 /app/vendor/yiisoft/yii2/base/Application.php(384): yii\base\Application::run
#0 /index.php(13): null
```

### Steps to reproduce

1. Deactivate the event
2. Make some changes to ticket types
3. Try to apply the changes

### Craft CMS version

5.5.5

### Plugin version

3.0.3

### Multi-site?

YES

**Comment by mihob** [comment]
This is probably a similar problem as here https://github.com/verbb/events/issues/164

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/events:"dev-craft-5 as 3.0.3"`.

**Comment by engram-design** [comment]
Fixed in [3.0.4](https://github.com/verbb/events/releases/tag/3.0.4)
